### PR TITLE
Remove any pseudo Basic auth handling because the server will perfor…

### PR DESCRIPTION
**Pull request by michael-o**
…m auth

### CODE REVIEW

File: include/setup.php [hunk 1]
```diff
@@ -570,7 +570,6 @@ function checkSendingAuthHeader($rep = false) {
 	}
 	$loggedin = $auth->hasUsername();
 	/*if (!$loggedin) {
-		header('WWW-Authenticate: Basic realm="'.str_replace('"', '\"', $auth->getBasicRealm()).'"');
 		header('HTTP/1.x 401 Unauthorized', true, 401);
 	} else {*/
```
Comment (joubertredrat) [inline include/setup.php]: Consider remove commented code, is versioned anyway.

Comment (michael-o) [reply include/setup.php]: Makes sense!

**Comment by michael-o** [comment]
@joubertredrat done!

**Comment by joubertredrat** [comment]
thanks @michael-o

**Pull request merged**